### PR TITLE
Update to version 0.1.7

### PR DESCRIPTION
**Pull request by adament**

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks.

Checksum will need an update as well. Feel free to change the checksum to `sha256`. Details about how to compute it if it is not already known can be found [here]( https://github.com/conda-forge/staged-recipes/blob/723daf799210ef0e05e0535c6ac1de503c3d912e/recipes/example/meta.yaml#L7-L10 ).

**Comment by adament** [comment]
Thank you! I am sorry if I end up spamming your notification feed, this is my first attempt at updating a conda-forge package.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: yaml-{{ version }}.tar.gz
   url: http://pyyaml.org/download/libyaml/yaml-{{ version }}.tar.gz
-  md5: 5fe00cda18ca5daeb43762b80c38e06e
+  md5: 1abf45bd3a96374fa55ca63b32f9f2f9
```
Comment (jakirkham) [inline recipe/meta.yaml]: Could we please switch to `sha256`? Admittedly it was this way before your update, but it would be a better check to use.

**Comment by jakirkham** [comment]
On a different point, GitHub seems to not understand the email that you used in git is one associated with this account. Would suggest adding that email to your GitHub account (one can have multiple emails associated) or switching to a different email in git and creating a fresh commit with it.

**Review by jakirkham — APPROVED** [review]

**Comment by scw** [comment]
Everything looks good with this branch, can it be merged?

**Comment by ocefpaf** [comment]
@scw missed this one. Sorry. But this is solved by #5 

Thanks for the PR!

**Pull request closed without merging**